Trigger/Algo/include/Trigger/TrigMipReco.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file TrigMipReco.h
+ * @brief Trigger Calo MIP finding algorithm
+ * @author Christian Herwig, Michigan
+ */
+
+#ifndef HCALTPSELECTOR_H
+#define HCALTPSELECTOR_H
```

Comment:
```suggestion
#ifndef TRIGGER_TRIGMIPRECO_H
#define TRIGGER_TRIGMIPRECO_H
```
was a copy paste error here originally too